packages/usa-nav/src/styles/_usa-nav.scss

```diff
@@ -111,6 +111,7 @@ $expand-less-icon: map-merge(
   }
 
   .usa-search {
+    margin-top: units(2);
```

Comment:
**Issue (blocking):** It looks like this change causes an unintentional visual regression in the full header display for default and megamenu variants. The top margin pushes down the search bar and it is no longer vertically centered to match the nav buttons.

| | Develop | Working branch |
|--------|--------|--------|
| Default | ![image](https://github.com/user-attachments/assets/a581d4d2-2dc9-4e79-928d-63b05ed44708) | ![image](https://github.com/user-attachments/assets/4d878384-28a1-4008-9725-ac816a5dd7c1) |
| Megamenu | ![image](https://github.com/user-attachments/assets/2f917d12-de1d-43b9-b7a5-826633631f2a) | ![image](https://github.com/user-attachments/assets/cac966b5-3dda-4c15-a5e0-2514b325d8f6) | 

**Suggestion:** This may be resolved by removing the margin in the media query below (I haven't confirmed).

```diff
margin-top: units(2);
+ // Additional empty space before media query to scan easier
 @include at-media($theme-header-min-width) {
   margin-left: units(2);
+  margin-top: 0;
}

Reply:
lgtm! Resolved in [adb5ba9](https://github.com/uswds/uswds/pull/6037/commits/adb5ba91bd7b3e7008a72f8d8bc43c77f1e0e0fc)